src/System.Runtime.Loader/tests/AssemblyLoadContextTest.cs

```diff
@@ -91,13 +90,12 @@ public static void LoadFromAssemblyName_AssemblyNotFound()
         }
 
         [Fact]
-        [ActiveIssue(15101)]
         public static void LoadFromAssemblyName_ValidTrustedPlatformAssembly()
         {
             string originalDir = Environment.CurrentDirectory;
             try
             {
-                Environment.CurrentDirectory = AppContext.BaseDirectory;
+                Environment.CurrentDirectory = Path.GetDirectoryName(typeof(string).Assembly.Location);
```

Comment:
Prior to this change, wasn't current directory not only containing the tests but also FX binaries? However, now it will be no longer the test folder but the location of _Microsoft.NETCore.App\shared\<version>_ folder. Setting it to be that does not sound right to me.

What happens if this remains the location where test is executing from?